monkestation/code/modules/uplink/uplink_items/misc.dm

```diff
@@ -6,3 +6,20 @@
 	purchasable_from = ~(UPLINK_NUKE_OPS | UPLINK_CLOWN_OPS)
 	item = /obj/item/syndie_glue
 	cost = 2
+
+/datum/uplink_item/device_tools/neutered_borer_egg
+	name = "Neutered borer egg"
+	desc = "A borer egg specifically bred to aid operatives. \
```

Comment:
can't be bought from nukeop uplinks

Reply:
Intended, borers benefit the most by progressing as time passes. I dont think nukies should be able to buy em